### PR TITLE
Some MAT Docs

### PR DESCRIPTION
**Pull request by Steve2955**

### CODE REVIEW

File: ML-FELDMANN/README.md [hunk 1]
```diff
@@ -24,12 +24,192 @@ Wissensbasierte Systeme / Neuronale Netze
...
+
+## Aufgabe 3
+
+> Zeichnen Sie die konvexe Menge der Punkte $(1,1)$, $(1,3)$, $(2,3.5)$, $(3,5)$, $(2,2)$, $(2,3)$.
+
+### Aufgabe 3: Lösung
+
+1. Punkte zeichnen
+2. Äußere Punkte der Punktmenge mit einem geschlossenen Kantenzug umranden
+3. Profit!!!
```
Comment (RvNovae) [inline ML-FELDMANN/README.md:215]: 😆

**Comment by RvNovae** [comment]
> @RvNovae Any ideas about this?
> 
> image
> 
> Seems to be caused by this block:
> 
> ```
> $$
> \xi = \begin{bmatrix}
> 1 & -3 & 1 & -2 & 2 & -2 & 1 & -3 & 1 & -2 & 2 & -2 \\
> 1 & -5 & 3 & -2 & 4 & -1 & 1 & -5 & 3 & -2 & 4 & -1 \\
> 1 & -1 & 1 & -1 & 1 & -1 & 1 & -1 & 1 & -1 & 1 & -1 \\
> \end{bmatrix}
> $$
> ```

Habs: https://tex.stackexchange.com/a/436649

**Comment by Steve2955** [comment]
> > @RvNovae Any ideas about this?
> > image
> > Seems to be caused by this block:
> > ```
> > $$
> > \xi = \begin{bmatrix}
> > 1 & -3 & 1 & -2 & 2 & -2 & 1 & -3 & 1 & -2 & 2 & -2 \\
> > 1 & -5 & 3 & -2 & 4 & -1 & 1 & -5 & 3 & -2 & 4 & -1 \\
> > 1 & -1 & 1 & -1 & 1 & -1 & 1 & -1 & 1 & -1 & 1 & -1 \\
> > \end{bmatrix}
> > $$
> > ```
> 
> Habs: https://tex.stackexchange.com/a/436649

Hat funktioniert 😄 Vielen Dank!

**Review by nukerxy — APPROVED** [review]
Profit

**Review by RvNovae — APPROVED** [review]

**Pull request merged**